src/Yoti.Auth/ShareUrl/Extensions/TransactionalFlowExtensionBuilder.cs

```diff
@@ -1,8 +1,9 @@
 ﻿namespace Yoti.Auth.ShareUrl.Extensions
 {
     /// <summary>
-    /// Allows you to provide a non-null object representing the content to be submitted in the TRANSACTIONAL_FLOW extension.
-    /// The object will be mapped to a Json representation using Jackson ObjectMapper.
+    /// Allows you to provide a non-null object representing the content to be submitted in the
+    /// TRANSACTIONAL_FLOW extension. The object will be mapped to a Json representation using
+    /// Jackson ObjectMapper.
```

Comment:
Is `Jackson ObjectMapper` reference from SDK Java?

Reply:
I understood this to mean that the code parsing the extension would be using Jackson ObjectMapper, so would be SDK-independent. @bucky-boy are you able to confirm this?

Reply:
Spoke with Vince, you were right! I've removed it in 5a2b971

Reply:
Yes, sorry I hadn't seen your question.  ObjectMapper is Java specific.
Besides, there's no point including this kind of detail in the docs anyway.  We should remove this from the java docs, it's not helpful